tests/foreman/ui/test_navigation.py

```diff
@@ -0,0 +1,115 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Login UI
+
+@Requirement: Navigation
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: UI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import skip_if_bug_open, tier1
+from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators, menu_locators
+from robottelo.ui.session import Session
+
+
+class NavigationTestCase(UITestCase):
+    """Implements the navigation tests from UI"""
+
+    def page_objects(self):
+        return {
+            'Activation Key': self.activationkey,
```

Comment:
The function would be redundant as it'd form a new dict on each call, and method is even more excessive - separate func for each class instance. That doesn't make much sense for me as this dict is basically constant, no need in different copies of it as we won't modify it. Anyway that may be considered as a nitpick too.